systems/citybuilding.go

```diff
@@ -47,7 +49,8 @@ func (cb *CityBuildingSystem) New(w *ecs.World) {
 // Update is ran every frame, with `dt` being the time
 // in seconds since the last frame
 func (cb *CityBuildingSystem) Update(dt float32) {
-	buildKey := engo.Button{[]engo.Key{engo.Q}, "build"}
+	engo.Input.RegisterButton("build", engo.Q)
```

Comment:
I'm pretty sure you're only supposed to register the button once in your setup, not ever frame.